packages/contracts/contracts/plugin/examples/count/CounterV1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This wont' work well. It should have a different structure, where each field could hold description as well.  Probably a separate task @brickpop but we need to design this before publishing plugins to our plugin repos.